deploy/aws/variables.tf

```diff
@@ -14,12 +14,12 @@ variable "eks_name" {
 
 variable "eks_version" {
   description = "Kubernetes version to use for the EKS cluster."
-  default     = "1.12"
+  default     = "1.15"
```

Comment:
is it possible to use the platform default value? if users do not specify it explicitly, the default kubernetes version in AWS will be used.

Reply:
IMO，using `default` or `latest` value is not stable. It may work well current time but encounter some problem when aws platform decided to update the version like this time( psp problem).

Reply:
make senses